<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnPhoneticAlphabet</class>
 <widget class="QWidget" name="LearnPhoneticAlphabet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: none;
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_vowel">
     <property name="title">
      <string>元音</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_monophthong">
        <property name="styleSheet">
         <string notr="true">QLabel {
	font: 35pt;
	min-height: 100px;
}</string>
        </property>
        <property name="title">
         <string>单元音</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_shortVowel">
           <property name="title">
            <string>短元音</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_shortVowel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>[i]</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btn_shortVowel_last">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_shortVowel_start">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_shortVowel_next">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_longVowel">
           <property name="title">
            <string>长元音</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_longVowel">
              <property name="text">
               <string>[:i]</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btn_longVowel_last">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_longVowel_start">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_longVowel_next">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_diphthong">
        <property name="styleSheet">
         <string notr="true">QLabel {
	font: 35pt;
	min-height: 135px;
}</string>
        </property>
        <property name="title">
         <string>双元音</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_diphthong">
           <property name="text">
            <string>[:d3]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_diphthongs_last">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_diphthongs_start">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_diphthongs_next">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_consonant">
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 35pt;
	min-height: 115px;
}</string>
     </property>
     <property name="title">
      <string>辅音</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_voicelessConsonant">
        <property name="title">
         <string>清辅音</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_voicelessConsonant">
           <property name="text">
            <string>[i]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_voicelessConsonant_last">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_voicelessConsonant_start">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_voicelessConsonant_next">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_voicedConsonant">
        <property name="title">
         <string>浊辅音</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_voicedConsonant">
           <property name="text">
            <string>[i]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_voicedConsonant_last">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_voicedConsonant_start">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_voicedConsonant_next">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_otherConsonant">
        <property name="title">
         <string>其他辅音</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_otherConsonant">
           <property name="text">
            <string>[i]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_otherConsonant_last">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/上一个.svg</normaloff>:/images/上一个.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_otherConsonant_start">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/播放.svg</normaloff>:/images/播放.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_otherConsonant_next">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/下一个.svg</normaloff>:/images/下一个.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
